custom-vu/src/main/kotlin/jces1209/vu/action/CreateAnIssue.kt

```diff
@@ -57,12 +58,13 @@ class CreateAnIssue(
             .flatMap { driver.findElements(it) }
             .filter { it.isDisplayed }
             .forEach { it.click() }
```

Comment:
it looks like it could be an exception if there are several buttons
because issue modal is opened after the first click and the second click can't be executed
let's fix it in this PR
```suggestion
            .first()
            .click()
```